src/governance.ts

```diff
@@ -128,8 +134,54 @@ export class GovernanceCanister {
     return this.responseConverters.toListProposalsResponse(rawResponse);
   };
 
+  public createNeuron = async ({
+    stake,
+    principal,
+    ledgerCanister,
+  }: {
+    stake: ICP;
+    principal: Principal;
+    ledgerCanister: LedgerCanister;
+  }): Promise<void> => {
+    if (stake.toE8s() < E8S_PER_ICP) {
+      throw "Need a minimum of 1 ICP to stake a neuron";
+    }
+
+    const nonceBytes = new Uint8Array(randomBytes(8));
+    uint8ArrayToBigInt;
+    const nonce = uint8ArrayToBigInt(nonceBytes);
+    const toSubAccount = buildSubAccount(nonceBytes, principal);
+
+    // Note - the account identifier here is just an alias of string.  Something to change?
+    const accountIdentifierAsString = principalToAccountIdentifier(
```

Comment:
We should use `AccountIdentifier.fromPrincipal(..)` here. `principalToAccountIdentifier` should ideally be deleted (but not necessarily in this PR of course)

Reply:
I didn't know they both do the same. Does that mean remove the code below as well:

```
const accountIdentifier = AccountIdentifier.fromHex(
      accountIdentifierAsString
    );
```

And instead just do:

```
cosnt accountIdentifier = AccountIdentifier.fromPrincipal(principal);
```

Reply:
Exactly!